app/controllers/application_controller.rb

```diff
@@ -7,4 +7,14 @@ class ApplicationController < ActionController::Base
 
   responders :flash
   respond_to :html
+
+  expose :serialized_current_user_data, -> { serialized_current_user_data }
```

Comment:
```suggestion
  expose :serialized_current_user_data, :serialized_current_user_data
```

Reply:
Does the second variant better then first?)